### PR TITLE
Add Quickstart to docs

### LINKED ISSUE #1481: Add a Getting Started/Quick Start page to the docs
We've gotten rid of the separate website to make maintaining the docs easier, but I think that means we need something similar in on Read The Docs, which is like the one page Quick Start guide that contains the information in https://github.com/weecology/retriever-site/blob/master/_posts/2016-02-27-quickstart.md.

### PR DESCRIPTION
**Pull request by ethanwhite**
Closes #1481

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+============
```
Comment (henrykironde) [inline docs/quickstart.rst:1]: Remove the extra =

Comment (ethanwhite) [reply docs/quickstart.rst]: Like the current change to make it the length of the word? (my rst is a little rusty :smile:)

Comment (henrykironde) [reply docs/quickstart.rst]: I didn't see this but that was what I was referring to thanks

**Comment by henrykironde** [comment]
LGTM

**Pull request merged**